include/deal.II/base/tensor.h

```diff
@@ -892,7 +892,11 @@ inline
 typename Tensor<rank_,dim,Number>::value_type &
 Tensor<rank_,dim,Number>::operator[] (const unsigned int i)
 {
-  return internal::TensorSubscriptor::subscript(values, i, internal::int2type<dim>());
+  // A fix for MSVC (prepending the 'dealii::')
+  // MSVC > C:\hpfem\agros2d\dealii\include\deal.II/base/tensor.h(886): error C2872: 'internal' : ambiguous symbol
+  // - could be 'dealii::internal'
+  // - or 'dealii::`anonymous-namespace'::internal'
+  return dealii::internal::TensorSubscriptor::subscript(values, i, dealii::internal::int2type<dim>());
```

Comment:
just drop the extra comment. This sort of information can be gotten from the git annotations, but in reality nobody will ever wonder why we need `dealii::` in some places.